### PR TITLE
Task  113

### PR DESCRIPTION
**Pull request by RanimHelamiah**

### CODE REVIEW

File: bootstrapers/showcase/templates/app/index.tsx [hunk 1]
```diff
@@ -98,7 +98,9 @@ export const AppTheme = ({
                     isAR={isAR}
                   />
                 ))}
-            <BCLink />
+            {config.whitlabel ? (
```
Comment (AMhaish) [inline bootstrapers/showcase/templates/app/index.tsx:101]: We should reverse the condition actually, the parameter should hide the BCLink, so by default we should always show that link unless a VIP user requested to hide it

**Comment by AMhaish** [comment]
We have some conflicts here can you please resolve them so we can continue merging

File: bootstrapers/showcase/templates/agency/index.tsx [hunk 2]
```diff
@@ -114,7 +114,9 @@ export const AgencyTheme = ({
                 isAR={isAR}
               />
             )}
-            <BCLink />
+            {!config.whitlabel ? (
+              null
+            ) : <BCLink />}
```
Comment (AMhaish) [inline bootstrapers/showcase/templates/agency/index.tsx:119]: I think still he has a problem here
config.whitlabel == true => we should not how BCLink
config.whitlabel  == false or undefined => we should show BCLink

Comment (RanimHelamiah) [reply bootstrapers/showcase/templates/agency/index.tsx:119]: Sorry for the confusion

**Pull request merged**